.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,79 @@
+name: 🐞 Bug, problem, error
+description: Fill a bug report here
+title: "Bug located in ..."
+labels: ["bug"]
+assignees: [""]
+
+body:
+
+  - type: markdown
+    attributes:
+      value: '# 🐞 **Problem**'
+
+  - type: checkboxes
+    id: new-bug
+    attributes:
+      label: Before submitting the issue
+      description: Please, make sure the following conditions are met
+      options:
+      - label: I have checked for [Compatibility](https://dpfdocs.pyansys.com/getting_started/index.html#compatibility) issues
```

```suggestion
      - label: I have checked for [Compatibility](https://dpf.docs.pyansys.com/getting_started/index.html#compatibility) issues
```
Sorry, just noticed we should use the new url now. redirection will only last so long.

---

Sure, I will make this change also to dpf-core then.